### PR TITLE
add Prisme Analytics

### PR DESCRIPTION
**Pull request by negrel**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Review by kokomo123 — APPROVED** [review]
LGTM, no issues here. Pending other thoughts from collaborators, thank you for your contribution!!! 💯

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]

File: software/prisme-analytics.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: "Prisme Analytics"
+website_url: "https://www.prismeanalytics.com"
+source_code_url: "https://github.com/prismelabs/analytics"
+description: "An Open Source, privacy-focused and progressive analytics service."
```
[the remark this answers is not in the document]
Comment (negrel) [reply software/prisme-analytics.yml]: done

File: software/prisme-analytics.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+licenses:
+  - AGPL-3.0
+  - MIT
+platforms:
+  - Go
+  - Docker
```
[the remark this answers is not in the document]
Comment (negrel) [reply software/prisme-analytics.yml]: done

File: software/prisme-analytics.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+tags:
+  - Analytics
+depends_3rdparty: false
```
[the remark this answers is not in the document]
Comment (negrel) [reply software/prisme-analytics.yml]: done

File: software/prisme-analytics.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+source_code_url: "https://github.com/prismelabs/analytics"
+description: "A privacy-focused and progressive analytics service based on Grafana."
+licenses:
+  - AGPL-3.0
+  - MIT
+platforms:
+  - Docker
+tags:
+  - Analytics
+  - Monitoring
```
Comment (nodiscc) [inline software/prisme-analytics.yml:12]:
```suggestion
```

```
ERROR:awesome_lint.py: 1 items tagged Monitoring, each tag must have at least 3 items attached
ERROR:awesome_lint.py: Prisme Analytics: tag Monitoring points to a tag which redirects to another list.
```

Monitoring tools are listed at https://github.com/awesome-foss/awesome-sysadmin?tab=readme-ov-file#monitoring. Analytics is enough.

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by nodiscc — APPROVED** [review]

**Review by Rabenherz112 — APPROVED** [review]

**Pull request merged**